src/icmp/pinger.cc

```diff
@@ -212,6 +209,7 @@ main(int, char **)
         }
 
         FD_SET(squid_link, &R);
+        Stopwatch timer;
```

Comment:
Odd. I used the github UI to accept your change. Not sure why the change disappeared again.

Reply:
In most cases, this happens when one force-pushes the PR branch after forgetting to "git pull" GitHub-committed changes. 

It is best to avoid force-pushing PR branches when possible. In most cases, it is possible.